content/en/blog/2021/proxyless-grpc/index.md

```diff
@@ -0,0 +1,400 @@
+---
+title: "gRPC Proxyless Service Mesh"
+description: Introduction to Istio support for gRPC's proxyless service mesh features.
+publishdate: 2021-09-02
+attribution: "Steven Landow (Google)"
+---
+
+Istio dynamically configures its Envoy sidecar proxies using a set of discovery APIs, collectively known as the
+[xDS APIs](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/operations/dynamic_configuration).
+These APIs aim to become a [universal data-plane API](https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a?gi=64aa2eea0283).
+The gRPC project has started to implement the xDS APIs, which means you can manage gRPC workloads without having to deploy
+an Envoy sidecar along with them. You can learn more about the integration in a [KubeCon EU 2021 talk from Megan Yahya](https://www.youtube.com/watch?v=cGJXkZ7jiDk).
+
+Istio 1.11 adds experimental support for adding gRPC services directly to the mesh. We support basic service
+discovery, some VirtualService based traffic policy, and mutual TLS.
+
+## Supported Features
+
+The current implementation of the xDS APIs within gRPC is limited in some areas compared to Envoy. The following
+features should work, although this is not an exhaustive list and other features may have partial functionality:
+
+* Basic service discovery. Your gRPC service can reach other pods and virtual machines registered in the mesh.
+* [`DestinationRule`](/docs/reference/config/networking/destination-rule/):
+    * Subsets: Your gRPC service can split traffic based on label selectors to different groups of instances.
+    * The only Istio `loadBalancer` currently supported is `ROUND_ROBIN`, `consistentHash` will be added in the future.
```

Comment:
consistentHash was just released in gRPC in case that is what is blocking the support.

Reply:
`consistentHash` is mostly blocked on our side

Reply:
Updated to reflect gRPC supports but istio doesn't